c2corg_api/views/document_changes.py

```diff
@@ -0,0 +1,160 @@
+import datetime
+from operator import or_, and_
+
+from c2corg_api.models import DBSession
+from c2corg_api.models.document import Document, ArchiveDocumentLocale
+from c2corg_api.models.document_history import DocumentVersion, HistoryMetaData
+from c2corg_api.models.user import User
+from c2corg_api.views import cors_policy
+from c2corg_api.views.feed import get_params
+from c2corg_api.views.validation import validate_token_pagination,\
+  validate_user_id_not_required
+
+from sqlalchemy.sql.expression import desc
+
+from cornice.resource import resource, view
+from sqlalchemy.orm import joinedload, load_only
+from urllib import parse as urllib_parse
+
+
+@resource(path='/document/changes', cors_policy=cors_policy)
```

Comment:
Please use ``documents`` instead of ``document``, to be consistent with the other service using ``waypoints``, ``routes``.
I think we use ``document`` for history service but it should be changed, as discussed this week with @tsauerwein .